docs/services.md

```diff
@@ -480,6 +483,38 @@ const staticImageUrl = request.url();
 // Now you can open staticImageUrl in a browser.
 ```
 
+```javascript
+// Filter all buildings that have a height value that is less than 300 meters
+const request = staticClient
+  .getStaticImage({
+    ownerId: 'mapbox',
+    styleId: 'streets-v11',
+    width: 200,
+    height: 300,
+    position: 'auto',
+    setfilter: [">","height",300],
+    layer_id: 'building',
+  });
+const staticImageUrl = request.url();
+// Now you can open staticImageUrl in a browser.
+```
+
+```javascript
+// Paint all the state and province level boundaries associated with the US worldview with a dashed line and insert it below the road-label layer
+const request = staticClient
+  .getStaticImage({
+    ownerId: 'mapbox',
+    styleId: 'streets-v11',
+    width: 200,
+    height: 300,
+    position: 'auto',
+    addlayer: {"id":"better-boundary","type":"line","source":"composite","source-layer":"admin","filter":["all",["==",["get","admin_level"],1],["==",["get","maritime"],"false"],["match",["get","worldview"],["all","US"],true,false]],"layout":{"line-join":"bevel"},"paint":{"line-color":"%236898B3","line-width":1.5,"line-dasharray":[1.5,1]}},
+    layer_id: 'road-label',
```

Comment:
`layer_id` is for `setfilter`, and I believe this example would actually return a 422 response from the API
```suggestion
    before_layer: 'road-label',
```